extract/omie_extractor.py

```
@@ -0,0 +1,417 @@
+                            print(f"✅ Successfully saved raw diario data for {day_str}")
+
+                            duplicates_df = pd.concat([duplicates_df, duplicates_df_day])
+
+            except Exception as e:
+                error_msg = f"Error downloading diario data for {day_str}: {e}"
+                print(f"  ❌ {error_msg}")
+                return duplicates_df, error_msg
+            
+        return duplicates_df, None
+
+    def extract_omie_intra(self, fecha_inicio_carga: Optional[str] = None, fecha_fin_carga: Optional[str] = None, 
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)